_posts/2019/2019-04-02-how-to-easily-manage-ssh-keys-part-2.md

```diff
@@ -0,0 +1,177 @@
+---
+layout: post
+title: how to easily manage ssh keys - part 2
+date: 2019-03-31
+header_image: public/ssh-key-management.jpg
+header_position: bottom
+header_overlay: true
+category: tech-stories
+tags: ["idoit", "ssh", "development", "tools"]
+authors: ["Carsten"]
+about_authors: ["cseeger"]
+---
+
+## Implementing the backend
+
+In the last part we created our information storage. Now its time to implement the backend.
+
+You may read my other blogposts and recognized that im a big fan of [Golang](https://golang.org/){:target="_blank"} and both the API and frontend is written in it.
+Also i've written and maintain a [client library impementation](https://github.com/cseeger-epages/i-doit-go-api){:target="_blank"} for i-doit in golang.
```

Comment:
Is this a relevant link for this post?

Reply:
im using the librarys in the code thats why i linked them.